test/python/quantum_info/test_synthesis.py

```diff
@@ -114,7 +114,7 @@ class TestOneQubitEulerDecomposer(QiskitTestCase):
 
     def check_one_qubit_euler_angles(self, operator, basis='U3',
                                      tolerance=1e-12,
-                                     phase_equal=False):
+                                     phase_equal=True):
```

Comment:
Can you leave the changes to `one_qubit_decompose()` for future? I think we should do all synthesis algorithms together (2Q too, as well as a bunch of things inside `qiskit.extensions`). I have a branch for this but was waiting for the `phase` property to exist on the circuit first.

Reply:
I'll see if I can backtrack while still passing tests.

Reply:
passes